app/forms/candidate_interface/gcse_missing_form.rb

```diff
@@ -1,10 +1,12 @@
 module CandidateInterface
   class GcseMissingForm
     include ActiveModel::Model
+    include GcseQualificationHelper
 
     attr_accessor :missing_explanation, :not_completed_explanation, :level, :subject, :qualification_type
 
-    validates :missing_explanation, word_count: { maximum: 200 }
+    validates :missing_explanation, word_count: { maximum: 50 }
+    validate :missing_explanation_presence
```

Comment:
Could we use the existing validation to check for the presence i.e. 
`validates :missing_explanation, presence: true, word_count: { maximum: 50 }`
And then set the error message in the yaml to override the generic "can't be blank"?

Reply:
Unfortunately not, because the name of the subject is in the error message and that's set dynamically depending on what it is.

Reply:
oh yeah, good idea. I was meaning to do this and then forgot. 

The structure is a bit funny, but that's the best I could do: https://github.com/DFE-Digital/apply-for-teacher-training/pull/8838/commits/29cbd8fde5679ee1dae89c82b2722e14a956146c

What do you think?